api/k8s/v1/k8s.proto

```diff
@@ -258,6 +258,25 @@ message Container {
   int32 restart_count = 5;
 }
 
+message PodCondition {
+  enum Type {
+    TYPE_UNSPECIFIED = 0;
+    CONTAINERS_READY = 1;
+    INITIALIZED = 2;
+    READY = 3;
```

Comment:
a super nit, to match the upstream field names
```suggestion
    POD_INITIALIZED = 2;
    POD_READY = 3;
```

Reply:
I was trying to match it to the strings defined in https://github.com/kubernetes/api/blob/v0.20.5/core/v1/types.go#L2520 so it's easier to compare. Does the meaning feel unclear?

Reply:
The `type PodConditionType` is actually a `string`

Reply:
But I see your point of matching it to the const defined here https://pkg.go.dev/k8s.io/api/core/v1#PodConditionType. Just thought that string comparison would be easier if https://github.com/lyft/clutch/pull/1226/files#diff-22775087d9b453b1fc3ce612415f660ea48a35823bab006fda020221bdaa3faeR96 maps to the exact string value

Reply:
oh I see! okay yeah I agree, the current name is good 👍